Index Settings Contributor sample

luanmaoski opened the pull request.

luanmaoski committed:
Add index settings contributor gradle sample

luanmaoski committed:
Add index settings contributor liferayworkspace sample

luanmaoski committed:
Add index settings contributor maven sample

luanmaoski committed:
Fix bnd

luanmaoski committed:
Fix bnb files

luanmaoski committed:
Add description for README.markdown

luanmaoski committed:
Add module to pom.xml

luanmaoski committed:
Fix checkSourceFormatting

luanmaoski committed:
Enhance README.markdown and fix settings.gradle

lrbladejenkins commented:
Source Formatting or Projects failed on LAX linux server on ffdb25b49008cae965a67b84c90a43ce912f5cd9
Please make sure to run
 
liferay-workspace:  gradlew checkSourceFormatting assemble 
maven: mvnw package

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1046/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1046/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on ffdb25b49008cae965a67b84c90a43ce912f5cd9

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/714/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/714/consoleText)

lrbladejenkins commented:
Source Formatting and Projects Built Successfully on LAX linux server on eab3531ae8930fe7b98ef4e647fa8530a3131b19

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1047/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1047/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on eab3531ae8930fe7b98ef4e647fa8530a3131b19

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/715/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/715/consoleText)

luanmaoski committed:
Add sample decription to README.markdown on project root

lrbladejenkins commented:
Source Formatting and Projects Built Successfully on LAX linux server on 40c3ae724140259b8fd56ae4bdc2708460c0033e

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1051/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1051/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on 40c3ae724140259b8fd56ae4bdc2708460c0033e

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/720/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/720/consoleText)

luanmaoski commented:
Hi @gamerson, these two commits [here](https://github.com/gamerson/liferay-blade-samples/pull/497/commits/474d056dabe58ad4a17be01d592f811d311bb8ca) and [here](https://github.com/gamerson/liferay-blade-samples/pull/497/commits/eab3531ae8930fe7b98ef4e647fa8530a3131b19) had to be done to make the `gradlew checkSourceFormatting assemble` pass.

When i run this command on my sample it works fine.

gamerson commented:
ci:test:71

lrbladejenkins commented:
Test failed on LAX linux server on 40c3ae724140259b8fd56ae4bdc2708460c0033e against latest 7.1 GA

[link to jenkins](http://cloud-10-50-0-99:8081/job/liferay-blade-samples-pr-tester-latest-ga-7.1/66/)

[Test Report](http://cloud-10-50-0-99:8081/job/liferay-blade-samples-pr-tester-latest-ga-7.1/66/testReport/)

[Console output](http://cloud-10-50-0-99:8081//job/liferay-blade-samples-pr-tester-latest-ga-7.1/66/consoleText)

lrbladejenkins commented:
Source Formatting or Projects failed on LAX linux server on 7585d2f3da8172bae79b09e4a16b0a7e9086a6dc
Please make sure to run
 
liferay-workspace:  gradlew checkSourceFormatting assemble 
maven: mvnw package

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1054/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1054/consoleText)

lrbladejenkins commented:
Source Formatting and Projects Built Successfully on LAX linux server on 58bb7a28aa0b301c6a84c32b31172e019a3f41f5

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1055/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1055/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on 7585d2f3da8172bae79b09e4a16b0a7e9086a6dc

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/723/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/723/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on 58bb7a28aa0b301c6a84c32b31172e019a3f41f5

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/724/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/724/consoleText)

luanmaoski commented:
ci:test:71

lrbladejenkins commented:
Test failed on LAX linux server on 58bb7a28aa0b301c6a84c32b31172e019a3f41f5 against latest 7.1 GA

[link to jenkins](http://cloud-10-50-0-99:8081/job/liferay-blade-samples-pr-tester-latest-ga-7.1/67/)

[Test Report](http://cloud-10-50-0-99:8081/job/liferay-blade-samples-pr-tester-latest-ga-7.1/67/testReport/)

[Console output](http://cloud-10-50-0-99:8081//job/liferay-blade-samples-pr-tester-latest-ga-7.1/67/consoleText)

lrbladejenkins commented:
Source Formatting and Projects Built Successfully on LAX linux server on c00560468cb0ed5f662a90c6a9c560b8e9a93668

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1057/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1057/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on c00560468cb0ed5f662a90c6a9c560b8e9a93668

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/726/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/726/consoleText)

luanmaoski commented:
ci:test:71

lrbladejenkins commented:
Test failed on LAX linux server on c00560468cb0ed5f662a90c6a9c560b8e9a93668 against latest 7.1 GA

[link to jenkins](http://cloud-10-50-0-99:8081/job/liferay-blade-samples-pr-tester-latest-ga-7.1/68/)

[Test Report](http://cloud-10-50-0-99:8081/job/liferay-blade-samples-pr-tester-latest-ga-7.1/68/testReport/)

[Console output](http://cloud-10-50-0-99:8081//job/liferay-blade-samples-pr-tester-latest-ga-7.1/68/consoleText)

lrbladejenkins commented:
Source Formatting and Projects Built Successfully on LAX linux server on 58bb7a28aa0b301c6a84c32b31172e019a3f41f5

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1058/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1058/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on 58bb7a28aa0b301c6a84c32b31172e019a3f41f5

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/727/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/727/consoleText)

luanmaoski commented:
Hi @gamerson, when i ran `gradlew testIntegration` inside "liferay-workspace" i got this error `org.jboss.arquillian.container.spi.client.container.LifecycleException: Error connecting to Karaf MBeanServer: `

Since this error isn't related to my sample, could you merge it?

Thanks.

gamerson commented:
yes @luanmaoski those errors are form the functional test and we don't have that for your sample yet, so I'll merge.

gamerson commented on liferay-workspace/apps/service-builder/jdbc/jdbc-service/build.gradle:
```diff
@@ -8,7 +8,7 @@ dependencies {
 	testCompile group: "biz.aQute.bnd", name: "biz.aQute.remote.api", version: "3.3.0"
 	testCompile group: "com.google.guava", name: "guava", version: "23.0"
-	testCompile group: "com.liferay.arquillian", name: "com.liferay.arquillian.arquillian-container-liferay", version: "1.0.6"
+	testCompile group: "com.liferay", name: "com.liferay.arquillian.extension.junit.bridge", version: "1.0.6"
```
I don't think we can make these changes, even though I know it is breaking SF.  @lawrence-lee   how did these pass SF in the past?  Perhaps we need to add a exclusion?

lawrence-lee replied:
they mustve made SF even more strict. It used to not care about the arquillian package inside build.gradle

gamerson commented on liferay-workspace/apps/service-builder/jndi/jndi-service/src/testIntegration/java/com/liferay/blade/samples/jndiservicebuilder/test/ConnectionTest.java:
```diff
@@ -58,7 +74,8 @@ public void testRegionService() throws Exception {
 	private static void _addData() throws Exception {
 		_executeNonQuery(
-			"create table region(id bigint not null primary key, name varchar(255));");
+			"create table region" +
```
This looks like a dangerous change made by SF.

gamerson replied:
But I guess it knows what it is doing?

lawrence-lee replied:
This file shouldve been ignored for SF. Is this rebased from latest?

lawrence-lee replied:
https://github.com/liferay/liferay-blade-samples/blob/7.1/liferay-workspace/apps/service-builder/jndi/jndi-service/source-formatter.properties

luanmaoski committed:
Revert changes service-builder/jdbc and service-builder/jndi

luanmaoski committed:
fix source formatting issues

lrbladejenkins commented:
Source Formatting or Projects failed on LAX linux server on d669a4de7c62892c279b5def232261d3d1971f2f
Please make sure to run
 
liferay-workspace:  gradlew checkSourceFormatting assemble 
maven: mvnw package

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1075/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1075/consoleText)

lrbladejenkins commented:
Source Formatting or Projects failed on LAX linux server on 137080982758356c446814b31e3f2d7b5e430dce
Please make sure to run
 
liferay-workspace:  gradlew checkSourceFormatting assemble 
maven: mvnw package

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/1076/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/1076/consoleText)

lrbladejenkinswindows commented:
Projects Built Successfully on LAX windows server on d669a4de7c62892c279b5def232261d3d1971f2f

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/744/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/744/consoleText)

luanmaoski commented:
Hi @gamerson, i rebased my branch with master and reverted the changes made on `jdbc-service/build.gradle` and `ConectionTest.java`.

When i ran `diff.sh` i got a error that isn't related with my sample.

lrbladejenkinswindows commented:
Projects failed to build on LAX windows server on 137080982758356c446814b31e3f2d7b5e430dce
Please make sure to run
 
liferay-workspace:  gradlew assemble 
maven: mvnw package

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/745/)

[Console output](http://cloud-10-50-0-254:80//job/liferay-blade-samples-pr-tester/745/consoleText)

gamerson commented:
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/518. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
